src/hotspot/share/services/heapDumper.cpp

```diff
@@ -2302,7 +2250,6 @@ void VM_HeapDumper::work(uint worker_id) {
       LockedClassesDo locked_dump_class(&do_class_dump);
       ClassLoaderDataGraph::classes_do(&locked_dump_class);
     }
-    Universe::basic_type_classes_do(&do_basic_type_array_class_dump);
```

Comment:
If this is the last use of this function, you could remove this variant of this function from Universe.

Reply:
It's still used in JvmtiGetLoadedClasses::getClassLoaderClasses()

Reply:
> void Universe::basic_type_classes_do(void f(Klass*)) {
This one is unused.  The other one that takes the Closure is the one that getClassLoaderClasses() calls.

Reply:
I thought we always have 2 version (for function pointer and closure object), Universe::metaspace_pointers_do has only version for closures.
Will remove unused one.